src/odemis/driver/smaract.py

```diff
@@ -4913,6 +4922,406 @@ def _cancelReference(self, future):
         return True
 
 
+class PicoscalePythonSDK(model.HwComponent):
```

Comment:
Let's not keep the Picoscale class, which is broken. I'd suggest you remove the old class and rename this class as `Picoscale`. The arguments are compatible with the old class, right? (so the microscope files don't need to be updated).

Reply:
> The arguments are compatible with the old class, right?

That's right

> I'd suggest you remove the old class and rename this class as `Picoscale`

I let the old class remain because it can be used a `fake` simulator. Should I create a fake simulator for the Python SDK?